lang/vi.json

```diff
@@ -63,7 +65,9 @@
     "Server Error": "Máy Chủ Gặp Sự Cố",
     "Service Unavailable": "Dịch Vụ Không Khả Dụng",
     "Showing": "Đang hiển thị",
+    "Tasks": "Công việc",
     "Task Create": "Tạo công việc",
+    "Task Detail": "Chi tiết công việc",
     "Task List": "Danh sách công việc",
     "Task Edit": "Chỉnh sửa công việc",
     "Thanks for signing up! Before getting started, could you verify your email address by clicking on the link we just emailed to you? If you didn't receive the email, we will gladly send you another.": "Cảm ơn bạn đã đăng ký! Trước khi bắt đầu, bạn có thể xác minh địa chỉ email của mình bằng cách nhấp vào liên kết mà chúng tôi vừa gửi qua email cho bạn không? Nếu bạn không nhận được email, chúng tôi sẽ sẵn lòng gửi cho bạn một email khác.",
```

Comment:
Em nên tạo file en.json rồi đặt key cho nó. để tránh phải gọi full text ntn 
"thankForSigning" : "Thanks for signing up! Before getting started....................."
Rồi dùng key này bên blade {{ __("thankForSigning") }}